### PR TITLE
Cypress testing 2

### LINKED ISSUE #41: Finish Cypress Testing

### PR DESCRIPTION
**Pull request by BriBourassa**
Change Summary:
- Test user stories with Cypress 
- Refactor previous Cypress testing branch that tests the updated code

closes #41 and #42

### CODE REVIEW

File: cypress/e2e/main.cy.js [hunk 1]
```diff
@@ -2,124 +2,147 @@ describe('As a user, I should be able to start the game from the main page, clic
...
+      cy.get('form').within(() => {
+        cy.get('input[type="text"]').type('Fluffy')
+        cy.get('button').click()
+      })
+        .get('button').eq(1).should('have.text', 'START GAME').click()
     })
 
-    it('should display what day the user is on', () => {
-      cy.get('button').eq(1).should('have.text', 'START GAME').click()
+    it('should display what day the user is on and give the story\'s intro', () => {
```
Comment (jsahim) [inline cypress/e2e/main.cy.js:27]: This is beefy but super thorough!

**Review by jsahim — APPROVED** [review]
Looks great!

File: cypress/e2e/book.cy.js [hunk 2]
```diff
@@ -5,49 +5,58 @@ describe('Cat Fact Book', () => {
     cy.intercept('GET', 'https://meowfacts.herokuapp.com/', {
       fixture: 'cat-fact.json',
     });
+    cy.wait(3000)
```
Comment (qrispi) [inline cypress/e2e/book.cy.js:8]: Really interesting how the timing on this works

**Review by Jesusborjas006 — APPROVED** [review]

**Review by qrispi — APPROVED** [review]
Bri you are a rockstar for championing all this testing! This is seriously awesome work!

**Pull request merged**